pvnet_app/forecast_compiler.py

```diff
@@ -261,6 +261,22 @@ def compile_forecasts(self) -> None:
                             f'which is greater than 10% of the national capacity '
                             f'({self.national_capacity})')
 
+        # Convert from xarray to a scalar
+        max_national_forecast_val = da_abs_national.sel(output_label="forecast_mw").max().values
+
+        # Warn if forecast is above 30 GW
+        if max_national_forecast_val > 30_000:  # 30 GW in MW
+            self.log_info(
+                f"WARNING: National forecast exceeds 30 GW ({max_national_forecast_val/1e3:.2f} GW)."
+            )
+
+        # Hard fail if forecast is above 100 GW
+        if max_national_forecast_val > 100_000:  # 100 GW in MW
+            raise Exception(
+                f"Hard FAIL: The maximum of the forecast is above 100 GW! "
+                f"Forecast is {max_national_forecast_val/1e3:.2f} GW."
+            )
```

Comment:
This looks great. I wonder if there is a way to unit test this? these 3 checks could be refactored to a new function called `validate_forecast(national_forecast_values, national_capacity)` and then simple unit tests could be run. 
We could do this in a different PR / issue if you prefer